apps/web/lib/features/daily-plan/all-plans-modal.tsx

```diff
@@ -209,63 +227,9 @@ export const AllPlansModal = memo(function AllPlansModal(props: IAllPlansModal)
 										closeModal={handleCloseModal}
 									/>
 								) : (
-									<>
```

Comment:
@CREDO23 (CC: @Cedric921 ) why we removed that whole block of code here? What it was doing, why we don't need that anymore?

Reply:
It is a duplication, the person who worked on #3028 did it !
I think we should reuse existing code ! 
So, I deleted it, and I am fixing #3028 in another PR to reuse existing code ! @evereq